docs/DeveloperGuide.md

```diff
@@ -193,19 +197,22 @@ The following sequence diagram shows how the clear operation works:
 
 ![ClearSequenceDiagram](images/ClearSequenceDiagram.png)
 
+<div markdown="span" class="alert alert-info">:information_source: **Note:** The method Parser::parse is a simplification of the overall parsing sequence which was already covered in showcasing the execution of the delete method. As such, redundant parsing details are not covered here.
```

Comment:
Please put this on a new line if not it will appear weird

![image](https://user-images.githubusercontent.com/10459011/98539421-449e6680-22c7-11eb-8d2f-f35d1319fccf.png)